src/constants/books.ts

```diff
@@ -0,0 +1,66 @@
+export const BOOK_LIST = [
```

Comment:
What happened with the missing books here? You must add all of them. If some book has `image=null`, you must handle this case with a default or placeholder image in `Book` component, not delete it.

For the placeholder image, you can use `img_book_6.png` from this [Drive](https://drive.google.com/drive/u/0/folders/0B0bbTjmKxmYsdWRQX1hSd2ZRT1k) (also `@2x` and `@3x` because Mobile has 3 different asset sizes depending of the cellphone size).

Reply:
Alright, i didn't knew i had to do that. I'm gonna incorporate it now.

Reply:
No problem, we must update the training card with this